packages/ckeditor5-image/theme/image.css

```diff
@@ -61,6 +61,9 @@
 	& .image > figcaption.ck-placeholder::before {
 		padding-left: inherit;
 		padding-right: inherit;
+		white-space: nowrap;
+		overflow: hidden;
+		text-overflow: ellipsis;
```

Comment:
Can you leave a comment as to why we need this with a link to the issue? These things are easily lost and CSS does not come with tests that would explain the code.